### PR TITLE
Add stat tracker

### PR DESCRIPTION
**Pull request by s-espinosa**
* Add rubric and spec for Stat Tracker project.

### CODE REVIEW

File: module1/projects/stat_tracker/rubric.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+| **Version Control** | Students implement a code review process for their pull request in which each partner makes suggestions for improvements, and those improvements are then integrated into the PR before it is merged. | Students did not contribute equally to the project in terms of number of commits or lines of code. Commits include multiple pieces of functionality. Pull requests present but do not include comments or were merged by their author. | Students contributed roughly equally to the project in terms of number of commits and lines of code. Commits are made in small chunks of functionality. Students used pull requests as a collaboration tool by reviewing each other’s PRs, making comments, and never merging their own PRs. | Less than 10 commits. No pull requests. Code is not hosted on the master branch of a Github repository. |
```
Comment (memcmahon) [inline module1/projects/stat_tracker/rubric.md]: I think 2 and 3 need to be switched.

Comment (s-espinosa) [reply module1/projects/stat_tracker/rubric.md]: Good catch! 🎉

File: module1/projects/stat_tracker/rubric.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+layout: page
+title: Stat Tracker - Rubric
+---
+
+| Category | **4: Well Above Expectations** | **3: Meets Expectations** | **2: Below Expectations** | **1: Well Below Expectations** |
+| -------- | ------------------------------ | ------------------------- | ------------------------- | ------------------------------ |
+| **Functionality** | Completes all functionality in iterations 1 through 4 and either Iteration 5 or Iteration 6. | Completes all functionality in iterations 1 through 4. | Does not complete all functionality in iterations 1 through 4. | Has significant missing functionality. |
```
[the remark this answers is not in the document]
Comment (s-espinosa) [reply module1/projects/stat_tracker/rubric.md]: Think this makes sense. Making this change.

File: module1/projects/stat_tracker/iterations/league_and_season_statistics.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+These statistics are calculated
+
+| Method | Description | Return Value |
+| ------ | ----------- | ------------ |
+|`count_of_teams`| Total number of teams in the data. | Integer |
+| `best_offense` | Name of the team with the highest average number of goals scored per game across all seasons. | String |
+| `worst_offense` | Name of the team with the lowest average number of goals scored per game across all seasons. | String |
+| `best_defense` | Name of the team with the lowest average number of goals allowed per game across all seasons. | String |
+| `worst_defense` | Name of the team with the highest average number of goals allowed per game across all seasons. | String |
+| `highest_scoring_visitor` | Name of the team with the highest average score per game across all seasons when they are home. | String |
```
[the remark this answers is not in the document]
Comment (s-espinosa) [reply module1/projects/stat_tracker/iterations/league_and_season_statistics.md]: Yup!

File: module1/projects/stat_tracker/iterations/game_statistics.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+| Method | Description | Return Value |
+| ------ | ----------- | ------------ |
+|`highest_total_score`| Highest sum of the winning and losing teams' scores | Integer |
+|`lowest_total_score`| Lowest sum of the winning and losing teams' scores | Integer |
+|`biggest_blowout`| Highest difference between winner and loser| Integer |
+|`count_of_ties`| Number of ties across all seasons and games | Integer |
+|`most_popular_venue`| Name of the venue with the most total games played | String |
+|`least_popular_venue`| Name of the venue with the fewest total games played|  String |
+|`percentage_home_wins`| Percentage of time that a home team has won a game (rounded to the nearest 100th) | Decimal |
```
[the remark this answers is not in the document]
Comment (s-espinosa) [reply module1/projects/stat_tracker/iterations/game_statistics.md]: Yup! Like this.

**Pull request merged**